### ISSUE TITLE
Missing "make install" step confuses some builds

### ISSUE THREAD

**Issue by adammoody** [LLNL/scr-top#5]
While hooking scr-top into a particular application, the build for that particular application invokes cmake/make/make install directly.  This almost works with the scr-top CMakeLists.txt, but it throws an error since we don't have a "make install" step.  I might look at the app build to see if I can get it to avoid the "make install" part for scr-top.

For now, I hacked around this by adding something like the following to the bottom of CMakeLists.txt so that "make install" is defined in the Makefile that cmake generates:
```
INSTALL(FILE NOTICE DESTINATION share/scr)
```
Not sure if there is a better CMake'y way to define a NOP make install step.

**Comment by gonsie** [comment]
Your install step is pretty clever and I think that's the only real solution we'll get: some fake/empty install that doesn't do anything real.

In the [CMake documentation](https://cmake.org/cmake/help/latest/command/install.html) I found:

```
INSTALL(CODE "MESSAGE(\"Sample Message\")")
```

Which simply prints a message.

I'll implement this later (this is mostly a message to myself for how to fix it).

**Comment by gonsie** [comment]
fixed with #8.
